crates/astria-composer/src/executor/builder.rs

```diff
@@ -48,6 +48,7 @@ impl Builder {
         } = self;
         let sequencer_client = sequencer_client::HttpClient::new(sequencer_url.as_str())
             .wrap_err("failed constructing sequencer client")?;
+
```

Comment:
nit: don't think this file needs any changes?

Reply:
Fixed in b8d148f.